setup.cfg

```diff
@@ -15,7 +15,7 @@ width = 80
 show-source = True
 max-line-length = 100
 exclude = .git,.tox,dist,*egg,build,tools
-ignore = E712,W503
+ignore = E711,E712,W503
```

Comment:
Hmm I'm not sure we should disable it globally. Does SQLAlchemy accept the `is not None` construct? If it does we can use it, if not I think it's better to exclude that check on the specific line.